docs/examples/model_masonry_scene.py

```diff
@@ -0,0 +1,26 @@
+from compas.files import OBJ
+# from compas_model.viewer import ViewerModel
+from compas_model.elements import Block
+from compas_model.model import Model
+from compas.datastructures import Mesh
+
+# Read the OBJ file of Cross vault.
+obj = OBJ('data/cross_vault.obj')
+obj.read()
+
+# Create elements from meshes and add them to the model.
+model = Model()
+elements = []
+for name in obj.objects:
+    mesh = Mesh.from_vertices_and_faces(*obj.objects[name])
+    block = Block(closed_mesh=mesh)
+    elements.append(block)
+    model.add_element("my_block", block)
+
+
+
+from compas.scene import Scene
+
+scene = Scene(context=None)
+print(Scene.viewerinstance)
+scene.add(elements[0])
```

Comment:
i don't think it makes much sense to use a no-context scene when working with a model. the only point would be to print the hierarchy of nodes, but since the model has its own tree, there is no need to construct a scene for that...

Reply:
removed, will keep on rhino and blender and the viewers